x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -319,3 +322,25 @@ export interface AlertingIndexUIQueryParams {
   date_range?: string;
   filters?: string;
 }
+
+export interface GetDatasourcesResponse {
```

Comment:
Can you add the comment back on these that references the kibana issue? I will likely implement it early next week in Ingest and will then refactor the code here to use them since we should really not be defining our own API response interface for ingest APIs

Reply:
Are you fine if I instead link to the code from the issue?

Reply:
Sure, that's fine 😃
(FYI - the issues I referenced in the `todo` comment were public and not to private repos)